test/src/vfs_helpers.h

```diff
@@ -350,10 +372,23 @@ class SupportedFsLocal : public SupportedFs {
    * No-op
    *
    * @param config Configuration parameters
+   * @param error Error parameter
    * @return Status OK if successful
    */
   virtual Status prepare_config(tiledb_config_t* config, tiledb_error_t* error);
 
+  /**
+   * Returns Status upon setting up the associated
+   * filesystem's number of threads
+   *
+   * @param config Configuration parameters
+   * @param error Error parameter
+   * @param num_threads Number of threads
+   * @return Status OK if successful
+   */
+  virtual Status prepare_threads(
```

Comment:
same here: this looks unused. please remove it and any of its implementations in the child classes